### ISSUE TITLE
Error in mcmc on continuous character set

### ISSUE THREAD

**Issue by abc-xy** [NBISweden/MrBayes#135]
# What is the current observed behaviour?
```
Division 1 has 33 unique site patterns
Initializing conditional likelihoods
Using standard non-SSE likelihood calculator for division 1 (single-precision)
Error in command "Mcmc"
```

### What is the expected/wanted behaviour?
I would like to run a data set in order to get a topology based on continuous characters:
ntax = 84, nchar = 35
type of characters = continuous (i.e., measurements)

MrBayes reads through the file and the MrB block alright.
After `mcmc` the above error shows up.
(there were no problems running MrBayes on molecular and combined molecular-discrete-morphological character sets)

### How may we reproduce this bug?
```
#NEXUS

Begin DATA;
	Dimensions ntax=84 nchar=35;
	Format datatype=Continuous gap=- missing=? INTERLEAVE; 
	Matrix
        
TaxonA_NRM587763	0.63	20.59	27.19	0.43	97.28	0.63	1.01	116.46	1.15	45.51	0.68	0.44	0.61	0.07	0.01	54.29	18.90	15.86	3.04	4.18	1.58	0.00	0.00	3.65	2.11	0.40	119.50	1.46	19.33	4.37	0.60	4703.41	56.14	127.40	78.14
TaxonB_NRM587764	1.44	10.75	17.00	1.13	75.57	1.78	3.11	116.51	1.03	36.48	1.85	1.38	1.68	0.17	0.01	36.21	11.43	9.37	2.07	0.44	0.89	0.00	0.00	8.04	4.75	1.33	126.00	1.56	36.95	2.09	0.60	4956.57	74.88	94.49	127.50
TaxonC_NRM606008	1.22	31.71	36.83	0.97	79.38	1.59	1.61	90.06	1.67	92.24	1.66	1.03	1.58	0.08	0.01	85.25	21.65	17.97	3.68	3.31	1.06	0.00	0.00	5.94	1.97	1.03	97.00	2.14	51.55	4.60	0.76	5477.70	23.50	96.20	82.75
.
.
.
TaxonX_22042	0.34	1.29	3.38	0.26	99.68	0.41	0.69	176.25	0.26	3.10	0.43	0.30	0.39	0.04	0.01	0.94	2.13	0.74	1.39	58.49	88.64	1.37	1.61	1.71	0.72	0.27	218.00	0.27	3.01	2.25	0.27	4700.45	19.71	171.01	7.50
TaxonY_22065	0.41	2.14	4.16	0.32	99.05	0.51	0.78	20.50	0.25	2.85	0.53	0.34	0.48	0.05	0.01	0.93	1.90	0.91	0.99	90.97	84.22	1.33	1.38	2.56	1.19	0.36	75.00	0.29	5.73	1.55	0.31	5034.33	48.67	37.45	63.49
TaxonZ_22096	0.31	1.19	4.07	0.21	98.12	0.30	0.94	177.12	0.18	1.40	0.31	0.19	0.26	0.06	0.00	1.04	1.88	0.61	1.22	144.72	255.67	1.56	2.41	2.16	1.40	0.24	85.00	0.40	4.91	2.20	0.24	4642.75	48.35	19.21	24.15
;

End;
begin mrbayes;
set autoclose=yes nowarn=yes;
exclude 22-23;
lset applyto=(all) rates=gamma coding=inf;
prset applyto=(all) statefreqpr=fixed(equal);
unlink statefreq=(all) revmat=(all) shape=(all); 

mcmcp ngen=40000000 relburnin=yes burninfrac=0.25 printfreq=5000 temp=0.35 diagnfreq=50000 samplefreq=5000 nchains=4 savebrlens=yes;

End;
```

### Would you be able to compile and run MrBayes to test fixes to this bug?
  - [ ] Yes, I know how to use `git` and how to compile MrBayes.
  - [ ] No, sorry.
  - [x] Not sure, depends on instructions given

### What is the environment that you run MrBayes in?
  - Operating system (including variant and release):
MacOS Sierra 10.12.6
  - Version of MrBayes: 
`MrBayes v3.2.6 x64
`

**Comment by nylander** [comment]
The data type `continuous` is actually not (yet) supported in MrBayes! The documentation on this was unfortunately lacking. We will address this with appropriate error checking and reporting to users.
/Johan

**Comment by nylander** [comment]
Now on branch [feature/gh_135](https://github.com/NBISweden/MrBayes/tree/feature/gh_135)

**Comment by andkaha** [comment]
The code on branch [`features/gh_135`](https://github.com/NBISweden/MrBayes/tree/feature/gh_135) now disallows using `continuous` as data type.

Using the type will result in an error like the following:
```
      MrBayes currently does not allow the use of the 'Continuous' datatype
      Error when setting parameter "Datatype" (2)
```

**Comment by andkaha** [comment]
This is also connected to issue #47

**Comment by nylander** [comment]
Awaiting the merging of code in pull request #142

**Comment by andkaha** [comment]
This has been addressed by PR #142
